api/handler/handlers_test.go

```diff
@@ -85,7 +85,7 @@ func prepareHandlerContext(t *testing.T) *handlerContext {
 
 	layerCfg := &layer.Config{
 		Caches:      layer.DefaultCachesConfigs(zap.NewExample()),
-		AnonKey:     layer.AnonymousKey{Key: key},
+		GateKey:     key,
```

Comment:
It's supposed to be a feature, see #271. Although I don't quite understand it.

Reply:
On current `master` it doesn't work:
```
$ aws s3api create-bucket --bucket $BUCKET --endpoint http://localhost:9080 --no-sign-request

An error occurred (InternalError) when calling the CreateBucket operation (reached max retries: 0): We encountered an internal error, please try again.
```
In the gate logs:
```
2023-08-04T07:42:20.324+0400    error   handler/util.go:29      call method     {"status": 500, "request_id": "dc3da4ba-b5b0-42aa-800f-2dbb71b80ddc", "method": "CreateBucket", "bucket": "heh3", "object": "", "description": "couldn't get bearer token signature key", "error": "couldn't get box data from context"}
```

In any case, we have to sign requests. Many client calls requires `Signer` as an explicit parameter. I tend to thing doesn't mater what key

Reply:
It can be used for reads, I guess.